biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.tsx

```diff
@@ -103,20 +105,21 @@ export const InputPage = () => {
           <b>throughout the town</b> so that our buildings can have an immediate
           energy source when they have a spike in energy consumption.
         </p>
-        <div className="startDatePicker">
-          <label>Start Date</label>
-          <DatePicker
-            showPopperArrow={false}
-            selected={startDate}
-            onChange={(date: Date) => setStartDate(date)}
-          />
-        </div>
-        <div className="endDatePicker">
-          <label>End Date</label>
-          <DatePicker
-            showPopperArrow={false}
-            selected={endDate}
-            onChange={(date: Date) => setEndDate(date)}
+        <div className="datePicker">
```

Comment:
Nit: `datePicker` is a known component type, with a calendar and everything.  To avoid confusion, give this class another name like `dateSlider`